tests/system/test_bucket.py

```diff
@@ -79,6 +83,9 @@ def test_bucket_lifecycle_rules(storage_client, buckets_to_delete):
             is_live=False,
             matches_storage_class=[constants.NEARLINE_STORAGE_CLASS],
         ),
+        LifecycleRuleAbortIncompleteMultipartUpload(
+            age=42,
+        ),
```

Comment:
Just noticed that the system tests never tested updating lifecycle rules (except for clearing them). I think we can add a section testing lifecycle actions being set via bucket patch/update.

Reply:
PTAL